lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -115,6 +115,33 @@ def assertion
             end
           end
 
+          # :nodoc:
+          class InDeltaAssertion < BasicAssertion
+            MATCHERS = %i[
+              assert_in_delta
+              assert_not_in_delta
+              refute_in_delta
+            ].freeze
+
+            NODE_MATCHER_PATTERN = <<~PATTERN
```

Comment:
We [do it differently now](https://github.com/rubocop/rubocop-rspec/blob/f062f923ce0b246e62ee9d61064ee1fedb54a298/lib/rubocop/cop/rspec/rails/minitest_assertions.rb#L72), right? This needs an update?